src/gui/advancedsettings.cpp

```diff
@@ -271,12 +327,16 @@ void AdvancedSettings::loadAdvancedSettings()
         combo_iface.setCurrentIndex(i);
     }
     addRow(NETWORK_IFACE, tr("Network Interface (requires restart)"), &combo_iface);
+    // Network interface address
+    updateInterfaceAddressCombo(combo_iface.currentIndex());
```

Comment:
In fact qt allows for a slot to have fewer parameters than the signal. So for this function you can omit the int parameter.
link: https://stackoverflow.com/questions/18388283/connecting-signals-to-slots-with-less-params-allowed-in-qt